src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,51 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import java.time.Duration;
+
+public class LoginPage extends BasePage{
+    By submitButton = By.cssSelector("[type='submit']");
+    By emailField = By.cssSelector("[type='email']");
+    By passwordField = By.cssSelector("[type='password']");
+    By profileIcon  = By.xpath("//a[@title='View/edit user profile']");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```